packages/renderer/package.json

```diff
@@ -34,7 +36,7 @@
     "monaco-editor": "^0.50.0",
     "postcss": "^8.4.38",
     "postcss-load-config": "^6.0.1",
-    "svelte": "4.2.18",
+    "svelte": "5.0.0-next.164",
```

Comment:
[svelte@5.0.0-next.168](https://github.com/sveltejs/svelte/releases/tag/svelte%405.0.0-next.168)

Reply:
the issue is that for example 166 was not working so I prefer to have something first working and then we do the update

Reply:
Okey so https://github.com/containers/podman-desktop/issues/7824 should stay open after the merge of this PR

Reply:
@axel7083 not sure to follow why it should stay open. I don't see the difference if it's 164 or 166 or 167
it comes to dependabot upgrades after it has been merged.

Reply:
You said 166 was not compatible with current, so fix are still needed to be compliant with v5 ?

Reply:
but it goes into the usual 'update libs to their latest version' workflow managed by dependabot

Reply:
Okey